docs/querying/basics.md

```diff
@@ -28,24 +28,23 @@ evaluate to one of four types:
 * **String** - a simple string value; currently unused
```

Comment:
note to self: start another PR due to native histograms experimental feature where there's a new type